.circleci/config.yml

```diff
@@ -149,30 +159,38 @@ jobs:
           7z x $sdl2_tmp -o'C:\lib\sdl2'
           $SDL_DIR = 'C:\lib\sdl2\SDL2-2.0.12'
           $sdl2_tmp | Remove-Item
+          $boost_tmp = New-TemporaryFile | Rename-Item -NewName { $_ -replace 'boost_tmp$', '7z' } –PassThru
+          Invoke-WebRequest $boost_url -OutFile $boost_tmp 
+          7z x $boost_tmp -o'C:\lib\boost'
+          ls C:\lib\boost\boost_1_73_0
+          ls C:\lib\boost\
+          $BOOST_DIR = 'C:\lib\boost\boost_1_73_0'
+          $boost_tmp | Remove-Item
       - run: |
+          cd ~
+          git clone --branch develop --depth 1 https://github.com/OpenMS/OpenMS.git
+          cd ~/OpenMS
+          ((Get-Content -path src\openswathalgo\include\OpenMS\OPENSWATHALGO\ALGO\Scoring.h -Raw) -replace '#include <vector>',"#include <vector>`r`n#include <algorithm>") | Set-Content -Path src\openswathalgo\include\OpenMS\OPENSWATHALGO\ALGO\Scoring.h
```

Comment:
Can you elaborate on the reasoning for this?  Is this a bug in OpenMS?

Reply:
We have to add `#include <algorithm>` otherwise the compiler won't build OpenSwathAlgo, I have submitted an issue to the OpenMS team : [[Error] - Failed to compile with VS2019 MSVC 14.2](https://github.com/OpenMS/OpenMS/issues/5020)

Reply:
I just looked over the bug report in OpenMS.  If you don't mind helping to get the fix included in OpenMS so we do not need this workaround, I think that would be the cleanest solution.

Reply:
I have submitted a PR to get this resolved, once it's in place we can remove this line.